test/e2e/storage/csi_volumes.go

```diff
@@ -398,7 +403,14 @@ func testCSIDriverUpdate(f *framework.Framework, driver drivers.TestDriver) {
 		err = framework.DeletePodWithWait(f, cs, pod)
 
 		By("Finding CSI driver deployment")
-		ds, err := getDriverDaemonSet(f)
+		var namespace string
+		if driver.GetDriverInfo().InstalledNamespace != "" {
+			namespace = driver.GetDriverInfo().InstalledNamespace
+		} else {
+			namespace = f.Namespace.Name
+		}
+
+		ds, err := getDriverDaemonSet(cs, namespace)
```

Comment:
`f.Namespace.Name` is still used below to update the DaemonSet and to wait for the update.